src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.controller
+
+import lotto.model.LottoResult
+import lotto.model.PurchasedTickets
+import lotto.model.RandomTicketNumberGenerator
+import lotto.view.InputView
+import lotto.view.ResultView
+
+class LottoController {
+
+    companion object {
```

Comment:
Class with only one companion object could be considered to be declared as [object](https://kotlinlang.org/docs/object-declarations.html).
(Also for the `InputView`, `ResultView`)